nix/mobile/android/jsbundle/default.nix

```diff
@@ -32,6 +32,8 @@ stdenv.mkDerivation {
             "package.json" "yarn.lock"
             # build stat's images to check if they exist
             "resources/.*" "translations/.*"
+	    # JavaScript files are required for creating directives for reanimated worklets
+	    "js/.*"
```

Comment:
There is no such folder as `js` at the root of the repo.

Reply:
Can you address this?

Reply:
Just removed the folder from this config. Included this from @Parveshdhull 's changes on issue #13452. Seems, it's not needed anymore, but will test that.

My CLJS build is currently broken somehow, will let you know, once i get it up and running again

Reply:
Cool.